src/utils/query-category.ts

```diff
@@ -1,9 +1,10 @@
 import type { Category } from "../types/category"
 
 import { API_HOST } from "../constants/env"
+import { SiteKey } from "../types/site"
 
-export async function getCategoryList(): Promise<Category[]> {
-  const response = await fetch(`${API_HOST}/categories`, {
+export async function getCategoryList(site: SiteKey): Promise<Category[]> {
+  const response = await fetch(`${API_HOST}/categories?site=${site}`, {
```

Comment:
We use `?site` query parameter here to specify which site we are on, and to provide a stable identifier for caching.